src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -24,7 +24,7 @@ import {COLORS, FONTS, SIZES} from '../../../utils/theme';
 import {Context} from '../../../context/Store';
 import {RECENT_SEARCH_TERMS} from '../../../utils/cache/constant';
 import {colors} from '../../../utils/colors';
-import {fonts} from '../../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `fonts` and `normalizeFontSize` has been modified. Ensure that the `normalizeFontSize` function is correctly defined and exported in the `../../../utils/fonts` module.
```diff
- import {fonts} from '../../../utils/fonts';
+ import {fonts, normalizeFontSize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->